helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+
+        <record id="group_helpdesk_timesheet_readonly" model="res.groups">
```

Comment:
Doesn't have any implied_ids?

Reply:
I see that it is to make only the Helpdesk and Timesheet menus visible later

Reply:
You're right. 

The only problem is that for now is too restrictive (e.g. clicking on Many2one field link within a ticket, like team will fire a permissions error), but we can add readonly ACLs later